Moderation-Policy.md

```diff
@@ -331,24 +331,23 @@ remove resigning team member from respective permissions and private access.
 * [ljharb](https://github.com/ljharb) -
   **Jordan Harband** &lt;ljharb@gmail.com&gt;
 
-## Escalation of Issues
+## Escalation of issues
 
-Moderation issue disputes not involving a TSC voting member or Moderation Team
-member may be escalated to the TSC for review by tagging the
-original issue, pull request, or associated nodejs/moderation repository
-tracking issue with the `moderation-review` label. Any such Moderation action
-may be overturned through a TSC vote.
+Any code of conduct report, and any decision made by the moderation team, can
+be escalated to the [OpenJS Code of Conduct Team](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/COC_POLICY.md#escalation).
+
```

Comment:
ו
```suggestion

Upon request, any information regarding an escalated report requested by the OpenJS Code of Conduct team will be supplied to them by the moderation team after approval by the TSC.
```

Reply:
strong -1: the [escalation policy](https://github.com/openjs-foundation/cross-project-council/blob/main/conduct/COC_POLICY.md#escalation) already states

> Projects are expected to participate in good faith in the escalation process and to share all the information related to the incident report with the CoC Team.